src/test/java/tutorspet/ui/ColorTest.java

```diff
@@ -0,0 +1,22 @@
+package tutorspet.ui;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class ColorTest {
```

Comment:
I don't think there is a need to test this class. It is not necessary to enforce that the pre-defined colours map to a specific Hex Value.

Reply:
Perhaps it would make more sense to ensure that they do not map to each other's value than that they map to a specific value?